import java.util.Scanner;

class Alpha1
{
	void alpha1() {
		
	System.out.println("Exception Handling");
	try {
	Scanner sc = new Scanner(System.in);
	System.out.print("Enter first number to divide :: ");
	int num1 = sc.nextInt();
	
	
	System.out.print("Enter sercond number to divide :: ");
	int num2 = sc.nextInt();
	System.out.println("The result is :: "+(num1/num2));
	System.out.println("Error Occured");
	}
	catch(Exception E) {
		System.out.println(E);
	}
	System.out.println("Connection terminated");
	}
}
public class LaunchExe4 {

	public static void main(String[] args) {
		try {
			System.out.println("Main Method connection");
			Alpha1 a = new Alpha1();
			a.alpha1();
		}
		catch(Exception E) {
			System.out.println(E);
		}
	}

}
